packages/react-ui-core/src/List/List.js

```diff
@@ -76,19 +90,23 @@ export default class List extends PureComponent {
           className,
         )}
       >
-        {items.map((item, i) => (
-          <Item
-            {...itemProps}
-            highlight={highlightIndex === i}
-            selected={selectedIndex === i}
-            key={this.itemId(i)}
-            index={i}
-            data-tid={`list-item-${i}`}
-            {...props}
-          >
-            {item}
-          </Item>
-        ))}
+        {items.map((item, i) => {
+          const [children, itemProps] = this.renderItem(item)
+          return (
+            <Item
+              {...itemMetaProps}
+              data-tid={`list-item-${i}`}
+              {...props}
+              {...itemProps}
+              highlight={highlightIndex === i}
```

Comment:
I don't remember well what itemProps and props contain, but it was set up before so that props would override item props if there were duplicates and this sets it up the opposite way, was that on purpose?

Reply:
Yeah, this was on purpose. I wasn't seeing a reason why we were able to override these.